ts/config.ts

```diff
@@ -18,6 +18,9 @@ const DEFAULT_FETCH_TIMEOUT_MS = 3000;
 // default max retries for fetch
 const DEFAULT_FETCH_MAX_RETRIES = 3;
 
+// default timeout of fetch for calling the PagoPA SOAP APIs
+const DEFAULT_FETCH_PAGOPA_TIMEOUT_MS = 15000;
```

Comment:
Is this going to be enough? Based on my tests it should be, but @LsKoder did mention that it sometimes took north of a minute to get a response. Has that happened to you as well @cloudify ?

Reply:
over 1-2 seconds we will have to make some changes to the UI as the call will happen in the background without the user knowing what's happening